packages/components/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 -   `Button`: Convert to TypeScript ([#46997](https://github.com/WordPress/gutenberg/pull/46997)).
 -   `BoxControl`: Convert to TypeScript ([#47622](https://github.com/WordPress/gutenberg/pull/47622)).
 -   `QueryControls`: Convert to TypeScript ([#46721](https://github.com/WordPress/gutenberg/pull/46721)).
+-   `Toolbar`: Convert to TypeScript ([#47087](https://github.com/WordPress/gutenberg/pull/47087)).
```

Comment:
One last thing — could you just move this line up to the "Unreleased" section again?

Reply:
Thanks @mirka! This is done with [4b2dbd0](https://github.com/WordPress/gutenberg/pull/47087/commits/4b2dbd0381fbfc6985878e0f51bbf8d9de7a7787).